doctrine.rst

```diff
@@ -445,9 +445,9 @@ some basic validation tasks::
         {
             $product = new Product();
             // This will trigger an error: the column isn't nullable in the database
-            $product->setName(null);
+                ->setName(null);
```

Comment:
This won't work as on the previous line the statement was terminated with the semicolon.